ansible/group_vars/centos7-ice3.6

```diff
@@ -30,15 +30,13 @@ default:
 
 omero_user: "omero"
 omero_user_home_dir: "/home/{{ omero_user }}"
-virtualenv_path: "{{ omero_user_home_dir }}/omerowebvenv"
+virtualenv_path: "/opt/omero/web/venv"
```

Comment:
@jburel : thoughts on moving this to `venv3` to match the ansible role?

Reply:
I will open follow up PR for omero-install and omeroweb-install

Reply:
PRs opened:
* https://github.com/ome/omero-install/pull/217
* https://github.com/ome/omeroweb-install/pull/50